### PR TITLE
Maintenance mode authorization handler

### PR DESCRIPTION
**Pull request by ltseeley**

### CODE REVIEW

**Review by peterschwarz — DISMISSED** [review]

File: libsplinter/src/rest_api/auth/maintenance/routes/actix.rs [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This module provides the following endpoints:
+//!
+//! * `GET /authroization/maintenance` for checking if maintenance mode is enabled
```
Comment (vaporos) [inline libsplinter/src/rest_api/auth/maintenance/routes/actix.rs:17]: Typo - authroization

File: libsplinter/src/rest_api/auth/maintenance/routes/mod.rs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+const AUTHORIZATION_MAINTENANCE_READ_PERMISSION: Permission =
+    Permission::Check("authorization.maintenance.read");
+#[cfg(feature = "rest-api-actix")]
+const AUTHORIZATION_MAINTENANCE_WRITE_PERMISSION: Permission =
+    Permission::Check("authorization.maintenance.write");
+
+/// The `MaintenanceModeAuthorizationHandler` provides the following endpoints as REST API
+/// resources:
+///
+/// * `GET /authroization/maintenance` - Check if maintenance mode is enabled
```
Comment (vaporos) [inline libsplinter/src/rest_api/auth/maintenance/routes/mod.rs:38]: same typo

File: libsplinter/src/rest_api/auth/maintenance/routes/actix.rs (hunk 1 above)
Comment (ltseeley) [reply libsplinter/src/rest_api/auth/maintenance/routes/actix.rs]: Fixed

File: libsplinter/src/rest_api/auth/maintenance/routes/mod.rs (hunk 2 above)
Comment (ltseeley) [reply libsplinter/src/rest_api/auth/maintenance/routes/mod.rs]: Fixed

**Review by vaporos — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Pull request merged**